samples/skills/FunSkill/Joke/config.json

```
@@ -2,6 +2,7 @@
   "schema": 1,
   "description": "Generate a funny joke",
   "type": "completion",
+  "function_completion_enabled": true,
```

I think you missed one in the string changes:
```suggestion
  "function_calling_enabled": true,
```